pkg/apis/pingcap/v1alpha1/tikv_config.go

```diff
@@ -26,56 +26,7 @@ type TiKVConfig struct {
 	LogRotationTimespan string `json:"log-rotation-timespan,omitempty" toml:"log-rotation-timespan,omitempty"`
 	// +optional
 	PanicWhenUnexpectedKeyOrData *bool `json:"panic-when-unexpected-key-or-data,omitempty" toml:"panic-when-unexpected-key-or-data,omitempty"`
-	// +optional
```

Comment:
why these fields are removed?

Reply:
They are placed to a wrong place (should be under the `[server]` section)